packages/sync-actions/src/shipping-methods-actions.js

```diff
@@ -55,10 +55,19 @@ function actionsMapZoneRatesShippingRates(diff, oldObj, newObj) {
 
 export function actionsMapZoneRates(diff, oldObj, newObj) {
   const handler = createBuildArrayActions('zoneRates', {
-    [ADD_ACTIONS]: newZoneRate => ({
-      action: 'addZone',
-      zone: newZoneRate.zone,
-    }),
+    [ADD_ACTIONS]: newZoneRate => [
+      {
+        action: 'addZone',
+        zone: newZoneRate.zone,
+      },
+      ...(newZoneRate.shippingRates
```

Comment:
Can we extract this into an function defined outside? `addFixedShippingRates` or something?

Reply:
Ah sure 👍

Reply:
Done! dc7bc9a

Note: I decided to call it `addShippingRates` because we are going to add not just fixed but tiered too 👍